diablo/sql_templates/update_rds_sis_sections.template.sql

```diff
@@ -80,4 +80,16 @@ INSERT INTO sis_sections (allowed_units, course_name, course_title, deleted_at,
   WHERE t.term_id = {term_id} AND s2.section_id IS NULL AND s2.term_id IS NULL
 );
 
+-- Newly created course-instructor pairings should generate new opt-in entries if the instructor has enabled the opt_in_new_courses preference.
+INSERT INTO opt_ins (term_id, section_id, instructor_uid, created_at)
+SELECT {term_id} AS term_id, s.section_id AS section_id, s.instructor_uid AS instructor_uid, now() AS created_at
+  FROM sis_sections s
+  LEFT JOIN tmp_sis_sections t
+  ON s.section_id = t.section_id AND s.term_id = t.term_id AND s.instructor_uid = t.instructor_uid AND t.deleted_at IS NULL
+  LEFT JOIN user_preferences u
```

Comment:
@pauline2k Should this be a JOIN instead? I think LEFT JOIN would include instructors without a row in user_preferences. Those would then get filtered out in the WHERE clause, but it might be less efficient.